platform/viewer/src/studylist/StudyListRoute.js

```diff
@@ -22,39 +29,88 @@ import UserManagerContext from '../context/UserManagerContext';
 import WhiteLabellingContext from '../context/WhiteLabellingContext';
 import AppContext from '../context/AppContext';
 
-const { urlUtil: UrlUtil } = OHIF.utils;
+const {
+  urlUtil: UrlUtil,
+  getContentFromArrayMap: getContentFromUseMediaValue,
+} = OHIF.utils;
+
+const DEFAULT_FILTERS = {
+  studyDateTo: null,
+  studyDateFrom: null,
+  patientName: '',
+  patientId: '',
+  accessionNumber: '',
+  studyDate: '',
+  modalities: '',
+  studyDescription: '',
+  //
+  patientNameOrId: '',
+  accessionOrModalityOrDescription: '',
+  //
+  allFields: '',
+};
+const DEFAULT_SORT = {
+  fieldName: 'patientName',
+  direction: 'desc',
+};
+const DEFAULT_ROWS_PER_PAGE = 25;
+const DEFAULT_PAGE_NUMBER = 0;
+const FETCH_DEBOUNCE_TIME = 225;
+
+function parseFilterValues(filterValues) {
+  const parseDateProp = dateKey => {
+    if (filterValues[dateKey] && filterValues[dateKey] !== '') {
+      filterValues[dateKey] = moment(filterValues[dateKey]);
+    } else {
+      filterValues[dateKey] = DEFAULT_FILTERS[dateKey];
+    }
+  };
+
+  parseDateProp('studyDateTo');
+  parseDateProp('studyDateFrom');
+  parseDateProp('studyDate');
+}
+
+const getInitialStudyListSessionProps = () => {
+  return {
+    sort: DEFAULT_SORT,
+    filters: DEFAULT_FILTERS,
+    rowsPerPage: DEFAULT_ROWS_PER_PAGE,
+    pageNumber: DEFAULT_PAGE_NUMBER,
+  };
+};
```

Comment:
I think this makes sense as an object, but I don't know that we need to call them "Session Props"?
When the component loads, we can hydrate from session storage, but it's still just state.

Reply:
ok.. changing it to a better name